src/Env/EnvEnum.php

```diff
@@ -20,6 +20,7 @@
  */
 class EnvEnum
 {
+    const ACME = 'acme'; // This env is not meant to be used except for testing purposes.
```

Comment:
We should try to find a way to avoid making this env available outside of testing at time to fetch env templates remotely